### DISCUSSION TITLE
Why could'nt I create many mailbox on same custom domain ?

### DISCUSSION THREAD

**Discussion by uncle-buzz** [simple-login/app#1916] [Q&A]
I have attached my domain as a custom domain on my premium account, I created a 1st simplelogin mailbox linked to one of my real mailboxes, but I can't create any new simplelogin mailbox for other real mailboxes on the same domain, I just receive a red message telling I can't create this mailbox without any details, I hope this is due to a misconfiguration and not a real limitation !

**Comment by Wolf1098** [comment]
The app doesn't provide a mailbox by it self, its basically a fancy forwarder. you must have mailbox else where under a different maybe free address. but by it self simplelogin, doesn't provide hosting unless you go with protonmail. and even then you get a to only use domains through simple login, unless you irk protons mx records, by placing them below simplelogins, allowing you to create address on proton, that will primarily be routed by simple login, and proton itself if that fails, which I half expect to will fail as well but im not sure to which extent they integrate the protonmail with simplelogin
